elephant/asset.py

```diff
@@ -457,37 +442,38 @@ def mask_matrices(matrices, thresholds):
 
 def _stretched_metric_2d(x, y, stretch, ref_angle):
     r"""
-    Given a list of points on the real plane, identified by their absciss x
-    and ordinate y, compute a stretched transformation of the Euclidean
+    Given a list of points on the real plane, identified by their absciss `x`
+    and ordinate `y`, compute a stretched transformation of the Euclidean
     distance among each of them.
-    The classical euclidean distance d between points (x1, y1) and (x2, y2),
-    i.e. \sqrt((x1-x2)^2 + (y1-y2)^2), is multiplied by a factor
+    The classical euclidean distance `d` between points (x1, y1) and (x2, y2),
+    i.e. :math:`\sqrt((x1-x2)^2 + (y1-y2)^2)`, is multiplied by a factor
     .. math::
 
             1 + (stretch - 1.) * \abs(\sin(ref_angle - \theta)),
 
-    where \\theta is the angle between the points and the 45deg direction
-    (i.e. the line y=x).
+    where :math:`\theta` is the angle between the points and the 45deg direction
+    (i.e. the line `y=x`).
     The stretching factor thus steadily varies between 1 (if the line
     connecting (x1, y1) and (x2, y2) has inclination ref_angle) and stretch
-    (if that line has inclination 90 + ref_angle).
+    (if that line has inclination `90 + ref_angle`).
 
     Parameters
     ----------
-    x : numpy.ndarray
+    x : np.ndarray
         array of abscissa of all points among which to compute the distance
```

Comment:
And here "abscissas" or "abscissae". I did not check whether the same should be corrected in other locations in the code.